Makefile

```diff
@@ -76,12 +76,12 @@ output:
 		.;
 	rm -rf output && mkdir output && tar -xf output.tar -C output && rm -f output.tar
 
-test-up4-integration-docker:
-	docker-compose -f test/integration/infra/docker-compose.yml rm -fsv
-	COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=$(DOCKER_BUILDKIT) docker-compose -f test/integration/infra/docker-compose.yml build $(DOCKER_BUILD_ARGS)
-	docker-compose -f test/integration/infra/docker-compose.yml up -d
+test-up4-integration-docker: DOCKER_TARGETS=pfcpiface
+test-up4-integration-docker: DOCKER_TAG=integration
+test-up4-integration-docker: docker-build
```

Comment:
what is done here? I don't think I have seen a target repeated multiple times in a Makefile before.

Reply:
Just setting env variables and making `test-up4-integration-docker` depend on `docker-build`. See https://stackoverflow.com/a/15230658.

Reply:
ok, that's what I expected. Thanks for the link.